packages/components-core/package.json

```diff
@@ -41,11 +41,12 @@
     "postdocs": "node ../../scripts/process-readme-props.js"
   },
   "devDependencies": {
-    "@percy/cli": "^1.0.0-beta.68",
-    "@percy/storybook": "^4.3.4",
```

Comment:
Was the removal of the percy packages needed? They may not be needed ATM, but we'll be bringing Percy back before long. It's okay if they're causing issues.  Just hoping to avoid us spending half a day before realizing we regressed in the Percy settings when we go to bring it back online. ;)